client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/ClientClusterFailoverEventsTest.java

```diff
@@ -112,6 +120,23 @@ public void testEventReplayWithAndWithoutStateAfterFailover() {
 
    }
 
+   private static void stopServerAndWait(RemoteCacheManager rcm, List<HotRodServer> servers,
+                                         List<EmbeddedCacheManager> managers) throws Exception {
+      InetSocketAddress addr = (InetSocketAddress) getLoadBalancer(rcm).nextServer(null);
+      for (HotRodServer server : servers) {
+         if (server.getPort() == addr.getPort()) {
+            ChannelFactory cf = rcm.getChannelFactory();
+            Channel channel = cf.fetchChannelAndInvoke(addr, new NoopChannelOperation()).get(10, TimeUnit.SECONDS);
+
+            findServerAndKill(rcm, servers, managers);
```

Comment:
is it OK that this method is now called for each server instance?

Reply:
TBH, I am still in doubt if all this here is necessary, I am thinking of reverting. The test in this class fails rarely.